src/PostList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type Props = {
+  posts: Post[];
+};
+
+const PostsList: React.FC<Props> = ({ posts }) => (
```

Comment:
Ensure consistency in naming conventions; the filename is "PostList," while the component name is "PostsList." Exercise caution with pluralization and strive for uniformity in naming throughout your code.